src/app/page/UserProfile/UserOverview.jsx

```diff
@@ -29,6 +35,34 @@ const UserStatus = ({ profile, setProfile, view }) => {
     setProfile(_.cloneDeep(profile));
   }
 
+  const handleAddressSelect = () => {
+    const addrObj = autocomplete.getPlace();
+    const addr = addrObj.formatted_address;
+
+    if (addr) {
+      profile["address"] = addr;
+      setProfile(_.cloneDeep(profile));
+    }
+  };
+
+  const handleScriptLoad = () => {
+    const options = {
+      type: ["(cities)"],
+    };
+    const addrElem = document.getElementById("address");
+    const googleRef = !!window && !!window.google ? window.google : undefined;
+
+    if (googleRef !== undefined) {
+      autocomplete = new googleRef.maps.places.Autocomplete(addrElem, options);
+      autocomplete.setFields(["address_components", "formatted_address"]);
+      autocomplete.addListener("place_changed", handleAddressSelect);
+    }
+  };
+
+  if (loaded && !error) {
+    handleScriptLoad();
```

Comment:
Should probably do this all inside a `useEffect` so we can watch when loaded and error change as well remove that listener on unmount

Reply:
Addressed in 1ea575f